web/pdf_find_controller.js

```diff
@@ -166,6 +206,10 @@ class PDFFindController {
       ) {
         return;
       }
+      if (cmd === "finddiacriticmatchingchange") {
+        this._pageDiffs.length = 0;
```

Comment:
Unless I'm overlooking something obvious here, it doesn't actually seem that you're guaranteed to ever re-populate `this._pageDiffs` after it's been cleared here!?

Won't that cause bugs, if the only thing changing is the "Match Diacritics" setting (for the same search term)?